params/types/coregeth/chain_config.go

```diff
@@ -163,7 +163,7 @@ type CoreGethChainConfig struct {
 	EIP1706FBlock *big.Int `json:"eip1706FBlock,omitempty"`
 
 	// https://github.com/ethereum/EIPs/pull/2537: BLS12-381 curve operations
-	EIP2537FBlock *big.Int `json:"eip2315FBlock,omitempty"`
+	EIP2537FBlock *big.Int `json:"eip2537FBlock,omitempty"`
```

Comment:
This seems to be a mistype which I fixed. Correct me if it was intentionally

Reply:
You are correct, it was my mistake